src/types/data.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * 0x prefixed 20 bytes ethereum address encoded as hex string
+ */
+export type Address = string;
```

Comment:
For this and `DataHexString` you can use ``0x${string}`` instead of `string` to be more specific 👌